### PR TITLE
add order by to get product list

### PR DESCRIPTION
**Pull request by arnesuscharles**

### CODE REVIEW

File: docs/openapi.yaml [hunk 1]
```diff
@@ -120,6 +120,14 @@ paths:
                   schema:
                     type: integer
                     format: int32
+                - name: orderBy
+                  in: query
+                  schema:
+                    type: string
+                - name: orderAscDesc
```
Comment (paulusherdiyatma) [inline docs/openapi.yaml:127]: can we change it to `order `or `direction `mas?

Comment (paulusherdiyatma) [reply docs/openapi.yaml:127]: and please provide the example

Comment (arnesuscharles) [reply docs/openapi.yaml:127]: Noted mas, so instead of orderAscDesc, it will be direction ya?

File: service/service.go [hunk 2]
```diff
@@ -503,17 +504,29 @@ func (s *service) DeleteProductById(ctx context.Context, userId uuid.UUID, id uu
...
+		Pagination:           pagination,
+		StoreID:              params.StoreID,
+		ProductTypeId:        params.ProductTypeId,
+		IsIncludeDeactivated: params.IsIncludeDeactivated,
+		OrderBy:              params.OrderBy,
+		OrderAscDesc:         params.OrderAscDesc,
+	}
+
+	if products, pagination, err = s.productRepository.GetProductsByStoreId(ctx, getProductsByStoreIdRepoParams); err != nil {
+		if strings.Contains(err.Error(), ErrIncorrectSqlSyntax) || strings.Contains(err.Error(), ErrInvalidColumnName) {
```
Comment (paulusherdiyatma) [inline service/service.go:527]: could we move all code related to DB error in the repository so the service does not have anything related to specific DB?

Comment (arnesuscharles) [reply service/service.go:527]: Okay noted mas, will move the DB error to the repository

**Pull request closed without merging**